win_build_vs.bat

```diff
@@ -21,6 +21,7 @@ if [%1]==[] goto usage
 
 set builddir=%1
 set skiptests=OFF
+set runtests=OFF
```

Comment:
I would keep the default of running tests. This script is used locally as well, not just for CI.

Reply:
Updated in 1502d43415f55f955f3e10c1f50967e6dc98050c